README.md

```diff
@@ -8,10 +8,31 @@ The stack also creates a small EC2 instance (defined in ec2-test.tf) that will b
 
 ## Deployment
 
+This stack is meant to be consumed as a module in your existing terraform stack. You can consume it by using code similar to this:
+
+    ````
+    module "ekk_stack" {
+        source = "github.com/GSA/devsecops-ekk-stack"
+        s3_logging_bucket_name = "${var.s3_logging_bucket_name}"
+        es_kinesis_delivery_stream = "${var.es_kinesis_delivery_stream}"
+    }
+    ````
+
+...where the variables referenced above are defined in your terraform.tfvars file.
+
+Following the steps below will emulate this exact behavior. You must execute it from the test directory just below the terraform directory. The test consumes the stack as a module and deploys it, then sets up an EC2 instance that will install the aws-kinesis-agent and configure it to stream to the Kinesis Firehose delivery stream.
```

Comment:
Minor: I'd split the README by `Usage` and `Development` (or `Test`) headings, to more clearly delineate.